File: SyncKit/Classes/CoreData/QSManagedObjectContext.m
//
//  QSManagedObjectContext.m
//  Pods
//
//  Created by Manuel Entrena on 29/07/2016.
//
//

#import "QSManagedObjectContext.h"

@implementation QSManagedObjectContext

- (void)performBlock:(void(^)())block
{
    [self performBlockAndWait:block];
}

@end

File: SyncKit/Classes/CoreData/QSManagedObjectContext.swift
//
//  QSManagedObjectContext.swift
//  OCMock
//
//  Created by Manuel Entrena on 15/06/2019.
//

import Foundation
import CoreData

class QSManagedObjectContext: NSManagedObjectContext {
    
    override func perform(_ block: @escaping () -> Void) {
        performAndWait(block)
    }
}
